docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+> - Evaluate the model
+> - Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Prerequisites
+
+For a list of prerequisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Model Builder multiclass classification overview 
+
+This sample creates a C# .NET Core console application that categorizes the risk of health violations using a machine learning model built with Model Builder. You can find the source code for this tutorial at the [dotnet/machinelearning-samples](https://github.com/dotnet/machinelearning-samples) GitHub repository.
+
+## Create a console application
+
+1. Create a **C# .NET Core console application** called "RestaurantViolations".
+
+## Prepare and understand the data
+
+> The data set used to train and evaluate the machine learning model is originally from the [San Francisco Department of Public Health Restaurant Safety Scores](https://www.sfdph.org/dph/EH/Food/score/default.asp). For convenience, the dataset has been condensed to only include the columns relevant to train the model and make predictions. Visit the following website to learn more about the [dataset](https://data.sfgov.org/Health-and-Social-Services/Restaurant-Scores-LIVES-Standard/pyih-qa8i?row_index=0).
+
+[Download the Restaurant Safety Scores dataset](https://github.com/luisquintanilla/machinelearning-samples/raw/AB1608219/samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantScores.zip) and unzip it.
+
+Each row in the dataset contains information regarding violations observed during an inspection from the Health Department and a risk assessment of the threat those violations present to public health and safety.
+
+| InspectionType | ViolationDescription | RiskCategory |
+| --- | --- | --- |
+| Routine - Unscheduled | Inadequately cleaned or sanitized food contact surfaces | Moderate Risk |
+| New Ownership | High risk vermin infestation | High Risk |
+| Routine - Unscheduled | Wiping cloths not clean or properly stored or inadequate sanitizer | Low Risk |
+
+- **InspectionType**: the type of inspection. This can either be a first-time inspection for a new establishment, a routine inspection, a complaint inspection, and many other types.
+- **ViolationDescription**: a description of the violation found during inspection. 
+- **RiskCategory**: the risk severity a violation poses to public health and safety.
+
+The `label` is the column you want to predict. When performing a classification task, the goal is to assign a category (text or numerical). In this classification scenario, the severity of the violation is assigned the value of low, medium, or high risk. Therefore, the **RiskCategory** is the label. The `features` are the inputs you give the model to predict the `label`. In this case, the **InspectionType** and **ViolationDescription** are used as features or inputs to predict the **RiskCategory**.
```

Comment:
I think in other tutorials we capitalize "Features" and "Label"

Reply:
Not in this case. Here, the context is the term label / features, not the field names in the "Data" screen in Model Builder. In that section, I capitalize the name of the field in the form like it's done in the Model Builder screen.

Reply:
#resolved